tests/test-cases/predefined-variables/.gitconfig

```diff
@@ -1,3 +1,3 @@
 
 [remote "origin"]
-	url = git@gitlab.com/gcl/predefined-variables.git
```

Comment:
i am not sure we still need this file - with my latest pull request which got merged by @firecow - i changed it to use `git remote -v` instead of the file. hence that this file is overall deprecated. or at least those lines :) but please tell me if i am wrong regrading my assumption

Reply:
I'm about to remove all .gitconfig's from all test folders.
I just need to figure out a way to make my jest-when mock fallback to the default behavior of Utils.spawn
You wouldn't happen to know, how this is done, would ya @aepfli ?

```typescript
// This is pseudo
import {when} from "jest-when";

export function initSpawnMock(spawnMocks: {cmd: string; returnValue: any}[]) {
    const mock = jest.fn();

    for (const spawnMock of spawnMocks) {
        when(mock).calledWith(spawnMock.cmd, expect.any(String)).mockResolvedValue(spawnMock.returnValue);
    }

    when(mock).calledWith(expect.any(String), expect.any(String)).mockResolvedValue( !!! default old Utils.spawn here !!!!)

    return mock;
}
```

Reply:
i would go with a spy here - based on my java knowledge - i need to check how jest-when is working with a spy, but you normally want to have the original implementation, and just overwrite certain calls.

Reply:
```typescript
// This is pseudo
import {when} from "jest-when";
import {Utils} from "...";

export function initSpawnMock(spawnMocks: {cmd: string; returnValue: any}[]) {
    const mock = jest.spyOn(Utils, "spawn");

    for (const spawnMock of spawnMocks) {
        when(mock).calledWith(spawnMock.cmd, expect.any(String)).mockResolvedValue(spawnMock.returnValue);
    }

    return mock;
}
```
something like that - not tested pseudo code